frontend/javascripts/dashboard/advanced_dataset/dataset_action_view.js

```diff
@@ -143,9 +145,9 @@ class DatasetActionView extends React.PureComponent<Props, State> {
             <div className="text-danger">{dataset.dataSource.status}</div>
           </div>
         ) : null}
-        {dataset.isActive ? (
+        {managesDataset && dataset.isActive ? (
```

Comment:
I don't think this change is equivalent. Only the Edit Dataset and Reload functionality should be restricted to Dataset Mangers/Admins, but all users with access should be able to view and trace the datasets, right? To achieve the same behavior as before, the `managesDataset` should be added to the `dataset.isEditable` check.

Reply:
Thanks for paying attention, I accidentally move the change a few lines up before committing :see_no_evil: